cmd/contour/shutdownmanager.go

```diff
@@ -277,11 +288,12 @@ func registerShutdownManager(cmd *kingpin.CmdClause, log logrus.FieldLogger) (*k
 }
 
 // registerShutdown registers the envoy shutdown sub-command and flags
-func registerShutdown(cmd *kingpin.CmdClause, log logrus.FieldLogger) (*kingpin.CmdClause, *shutdownContext) {
-	ctx := newShutdownContext()
+func registerShutdown(cmd *kingpin.CmdClause, defaultAdminPort int, log logrus.FieldLogger) (*kingpin.CmdClause, *shutdownContext) {
+	ctx := newShutdownContext(defaultAdminPort)
```

Comment:
This should pass `ctx.adminPort` and not `defaultAdminPort` which should be removed.

Reply:
@stevesloka same thing here: the defaultAdminPort param passed is exactly the ctx.AdminPort which is passed to the registerShutdown function. So, what's the thing to be changed here?

Reply:
@stevesloka same thing here: the defaultAdminPort param passed is exactly the ctx.AdminPort which is passed to the registerShutdown function. So, what's the thing to be changed here?